Wrapping/TypedefMacros.cmake

```diff
@@ -171,6 +171,8 @@ macro(itk_end_wrap_module)
       "${ITK_WRAP_PYTHON_LIBRARY_CONFIG_FILE}"
       "${ITK_WRAP_PYTHON_SNAKE_CASE}"
     )
+    unset(ITK_WRAP_PYTHON_LIBRARY_CONFIG_FILE)
```

Comment:
A bunch of these small commits should be squashed together.

Reply:
Done.